src/mail/editor/SendMailModel.ts

```diff
@@ -557,7 +558,7 @@ export class SendMailModel {
 	 */
 	removeRecipient(recipient: Recipient, type: RecipientField, notify: boolean = true): boolean {
 		const recipients = this.recipients.get(type) ?? []
-		const didRemove = findAndRemove(recipients, (r) => r.address === recipient.address)
+		const didRemove = findAndRemove(recipients, (r) => cleanMailAddress(r.address) === cleanMailAddress(recipient.address))
```

Comment:
maybe we could at least pull `cleanMailAddress(recipient.address)` out?

Reply:
yes